core/Util/plJPEG.h

```diff
@@ -17,7 +17,8 @@
 #ifndef _PLJPEG_H
 #define _PLJPEG_H
 
-#include "PRP/Surface/plMipmap.h"
+#include "PlasmaDefs.h"
+#include "Debug/hsExceptions.hpp"
```

Comment:
```suggestion
```

This should probably be in the .cpp file.

Reply:
hsJPEGException is unhappy with that.